src/views/Auth/Auth.jsx

```diff
@@ -17,6 +17,16 @@ export default function Auth({ isSigningUp = false }) {
       // If signing in: set the user ({id, email}) and redirect to /notes
       // If signing up: redirect to /confirm-email
       // Use the corresponding functions from `/services/users` for both cases
+    if(isSigningUp) {
+      signUpUser(email, password);
+      history.replace('/confirm-email');
+    } 
+    if(signInUser) {
+      const user = await signInUser(email, password)
+      setUser(user);
```

Comment:
Remember that the context is looking for the user object to be the {id, email} so you'd want to reference those properties specifically when passing the user into the `setUser` function.